.github/workflows/py-ci.yml

```diff
@@ -91,18 +75,17 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       QUILT_DISABLE_USAGE_METRICS: true
+    defaults:
+      run:
+        working-directory: api/python
```

Comment:
probably that doesn't affect codecov action and you need to use [`directory`](https://github.com/codecov/codecov-action?tab=readme-ov-file#arguments)